src/selectors/entities/preferences.test.js

```diff
@@ -299,6 +309,34 @@ describe('Selectors.Preferences', () => {
             }).mentionBg, theme.mentionBg);
         });
 
+        it('updates sideBarTeamBarBg variable when its not present', () => {
+            const currentTeamId = '1234';
+            const theme = {sidebarHeaderBg: '#ff0000'};
+
+            assert.deepEqual(Selectors.getTheme({
+                entities: {
+                    general: {
+                        config: {
+                            DefaultTheme: 'default',
+                        },
+                    },
+                    teams: {
+                        currentTeamId,
+                    },
+                    preferences: {
+                        myPreferences: {
+                            [getPreferenceKey(Preferences.CATEGORY_THEME, '')]: {
+                                category: Preferences.CATEGORY_THEME, name: '', value: JSON.stringify({}),
```

Comment:
You can just define this value instead of making it empty and having a separate team-specific theme defined next. The two different preferences are only needed if you're testing team-specific themes.